src/htcondor_es/convert_to_json.py

```diff
@@ -971,10 +981,12 @@ def commonExitCode(ad):
     Consolidate the exit code values of JobExitCode, 
     the  chirped CRAB and WMCore values, and
     the original condor exit code.
+    JobExitCode and Chirp_CRAB3_Job_ExitCode
+    exists only on analysis jobs.
     """
-    return ad.get('Chirp_WMCore_cmsRun_ExitCode',
-                  ad.get('JobExitCode',
-                         ad.get('Chirp_CRAB3_Job_ExitCode',
+    return ad.get('JobExitCode',
```

Comment:
@lecriste this is what we should change to CRAB_PJ_JobExitCode, right ? It may be good that @cronosnull puts a comment in the meanwhile, since an ad named only JobExitCode is way too much confusing. It is quite clear that the exit code of a job is not an obvious things and we have several pieces of software competing in trying to set it

Reply:
Exactly.